pkg/healthautoexport/types.go

```diff
@@ -25,12 +28,26 @@ type PayloadData struct {
 
 // Metric defines a single measurement with units, as well as time-series data points.
 type Metric struct {
-	Name  string       `json:"name"`
-	Units Units        `json:"units"`
-	Data  []*Datapoint `json:"data"`
+	Name          string           `json:"name"`
+	Units         Units            `json:"units"`
+	Data          []*Datapoint     `json:"data"`
+	SleepAnalyses []*SleepAnalysis `json:"-"`
 }
 
-func (m Metric) GetUnits() Units {
+type SleepAnalysis struct {
```

Comment:
nit: Add godoc for this type in line with other types in this package.

Reply:
Fixed.